docs/pages/application-access/getting-started.mdx

```diff
@@ -22,8 +22,20 @@ Let's connect to Grafana using Teleport Application Access in three steps:
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
+
+<Tabs>
+<TabItem label="Docker">
 - A Docker installation, which we will use to launch Grafana in a container. Alternatively, if you have another web application you'd like to protect with Application Access, you can use that instead.
 - A host where you will run the Teleport Application Service.
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+
+(!docs/pages/includes/kubernetes-access/helm-k8s.mdx!)
+
+- Administrative rights to deploy within a cluster.
+
+</TabItem>
+</Tabs>
 
 <Admonition type="tip" title="Not yet a Teleport user?">
```

Comment:
Is this section made redundant by the `edition-prereqs-tabs.mdx` partial?

Reply:
@stevenGravy Looks like this is the last remaining piece of feedback on this PR. Would you mind taking a look? Thanks!